release_notes/v3.0.0-beta.md

```diff
@@ -0,0 +1,127 @@
+v3.0.0-beta Release Notes - March 14, 2024
+==========================================
+
+The v3.0.0-beta release is an early release of Fabric v3.0, specifically to demonstrate and get feedback on the new Byzantine Fault Tolerant (BFT) ordering service.
+
+v3.0.0-beta is not intended to be used in production and is not intended as an upgrade target from prior versions. It can however be used to test the process
+for migrating from Raft to SmartBFT consensus.
+
+New features
+------------
+
+**Byzantine Fault Tolerant (BFT) ordering service**
+
+Hyperledger Fabric has utilized a Raft crash fault tolerant (CFT) ordering service since version v1.4.1.
+A Byzantine Fault Tolerant (BFT) ordering service can withstand not only crash failures, but also a subset of nodes behaving maliciously.
+Fabric v3.0.0-beta provides a BFT ordering service based on the [SmartBFT](https://arxiv.org/abs/2107.06922) [consensus library](https://github.com/SmartBFT-Go/consensus).
+Consider using the BFT orderer if true decentralization is required, where up to and not including a third of the parties running the orderers may not be trusted due to malicious intent or being compromised.
+For more details of the BFT ordering service and other recent features, see the [What's New documentation](https://hyperledger-fabric.readthedocs.io/en/latest/whatsnew.html).
+
+The v3.0.0-beta release differs from the v3.0.0-preview release by making the peer and orderer block replication resistant to block withholding. 
+Block withholding is when an orderer receives a request for blocks it possesses, but it delays sending the blocks and by doing so, slows down the requester's block reception.
+As described in [the RFC](https://hyperledger.github.io/fabric-rfcs/text/0010-bft-signatures.html), block withholding resistance incurs a bandwidth and CPU cost,
+and therefore an operator of an orderer may opt out and use the standard mechanism by specifying `ReplicationPolicy`: `simple` in the local configuration.
```

Comment:
I searched for ReplicationPolicy. It is not mentioned in the docs, and the orderer.yaml entry is not described well. It probably deserves a better description in orderer.yaml, and perhaps a mention in the docs, if that is a major part of the beta release.

Reply:
i will do it, but, we can always update the docs post release, can't we?

Reply:
I updated the `orderer.yaml`, i hope it's more clear now.

Reply:
Yes, we can edit docs anytime. Just want to make sure anything we want to highlight in the beta is there. Merging.